### PR TITLE
Update reserved keywords in umbraco

### PR DESCRIPTION
**Pull request by Fjellvang**

### CODE REVIEW

**Comment by rasmusjp** [comment]
Can we please keep the ordering alphabetical? :) Also I think there's missing a couple in regards to the block list

File: Umbraco-Heartcore/API-Documentation/GraphQL/Schema-Generation/index.md [hunk 1]
```diff
@@ -221,6 +231,8 @@ List of reserved Content Type Property names, these cannot be used as a Property
 * sortOrder
 * updateDate
 * url
+* parentId
+* content
```
Comment (rasmusjp) [inline Umbraco-Heartcore/API-Documentation/GraphQL/Schema-Generation/index.md:235]:
```suggestion
* content
* parentId
* sortOrder
* updateDate
* url
```

File: Umbraco-Heartcore/API-Documentation/GraphQL/Schema-Generation/index.md [hunk 2]
```diff
@@ -163,6 +163,7 @@ List of reserved type names, these cannot be used as an `alias` for Content Type
 The GraphQL type name is the Content Type `alias` converted to Pascal Case.
 :::
 
+* BlockListItem
 * BigInt
```
Comment (rasmusjp) [inline Umbraco-Heartcore/API-Documentation/GraphQL/Schema-Generation/index.md:167]:
```suggestion
* BigInt
* BlockListItem
```

**Review by rasmusjp — APPROVED** [review]

**Comment by eshanrnh** [comment]
Looks good, @Fjellvang & @rasmusjp 💪 Merging it in 🙂

**Pull request merged**